tools/generator/AnDOGenerator.py

```diff
@@ -0,0 +1,117 @@
+import pandas as pd
+import argparse
+import os
+import sys
+
+# columns required
+COLUMNS = ["experiments_name", "subjects_names",
```

Comment:
variable name in capitals? is this PEP8 ok?

Reply:
![image](https://user-images.githubusercontent.com/17256769/100617639-14bd0e80-331b-11eb-9ec7-6604ae1c84bd.png)
From pep8 documentation

Reply:
thanks!